src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { name } = props;
+
+  return (
+    <button type="button">{name}</button>
+  );
+};
```

Comment:
[Optional]

Let's destructure `name` directly from the parameter.
```suggestion
const Button = ({ name }) => (
   <button type="button">{name}</button>
);
```